### ISSUE TITLE
The Legions of the Undead follow give giant snakes more rights then survivors.

### ISSUE THREAD

**Issue by Warbidon** [CleverRaven/Cataclysm-DDA#25785]
**Describe the bug**  
Giant Snakes will attempt to maul Zombie Soldiers, it may maul other zombies but I have only witnessed 1 particular snake mauling a soldier. It does not fight back, nor does it's fellow shamblers come to its assistance. The undead does not give me this mercy.

**To Reproduce**  
1. Acquire a Giant Snake, a Zombie Soldier, and other zombies.
2. Put them int he Death Arena.
3. Witness the Pacifist Zombies get mauled by a snake.

**Expected behavior**  
Either the giant zombie should ignore the zombies and slither on like the absolute unit of a snake it is, or it should expect the same violence it inflicts to be inflicted upon itself.

**Screenshots**  
https://i.imgur.com/FEtzBhF.png

**Versions and configuration(please complete the following information):**
 - OS: Windows 7
 - Game Version [from the main menu, e.g. 0.C-32238 - g8f0b634
 - Graphics version Terminal
 - Mods loaded: **Filthy Clothing**, **Disable NPC Needs**, **Craftable Gun Pack**, **Simplified Nutrition**, **Safe Autodoc** **Bright Nights** **Dark Days Ahead: Lua** (Fun, Items, Skills, Traits) **Bionic Systems**, **Makeshift Items**, **More Survival Tools**, **Mining Mod**, **Beta National Gaurd Camp**,  **Urban Development**, **Boats**, **Folding Parts Pack**, **Vehicle Additions Pack**, **Tanks And Other Vehicles Pack**, **Arcana and Other Magic Items**, **Necromancy** ((dont judge me pls))

**Comment by FulcrumA** [comment]
Probably at the very least the zombie should defend itself but otherwise remain oblivious to the animal. On the other hand according to the lore, creature of that size would make a suitable blob host so perhaps zombies should hunt them as well.

**Comment by Tatterdemalian** [comment]
Well, the RNG just decided to double-team my longest lived survivor with a soldier zombie and a giant rattlesnake. Couldn't outrun either, so I maneuvered them into fighting each other. What I noticed was, while the snake eagerly started biting the crap out of the soldier, it did no damage at all. Guess gear designed for field combat is nearly impenetrable to the teeth and claws of field monsters. Two hundred turns later, I managed to beat the zombie to death, one hit point at a time, with my nail bat, only to then be finished off by the snake.

Frustrating, but I don't think it's a bug that zombies don't notice cold blooded reptiles that can't hurt them.
